### PR TITLE
Clean up activity list descriptions on claims status

### PR DESCRIPTION
**Pull request by jbalboni**
Related to [46](https://github.com/department-of-veterans-affairs/sunsets-team/issues/46).

This updates the descriptions to match as much information as we understand from the evss data.
screen shot 2016-10-12 at 3 58 33 pm

### CODE REVIEW

**Comment by jbalboni** [comment]
@austin-martinez Can you take a look and see if the way I'm using the tracked item info still makes sense?

cc @cagedcrown

File: src/js/disability-benefits/components/ClaimPhase.jsx [hunk 1]
```diff
@@ -27,55 +27,89 @@ function getClasses(phase, current) {
...
+        );
+      case 'filed':
+        return <div className="claims-evidence-item columns medium-9">Thank you. VA received your claim</div>;
+      case 'completed':
+        return <div className="claims-evidence-item columns medium-9">Your claim is complete</div>;
+      default:
+        break;
+    }
+
+    if (event.type === 'still_need_from_you_list' || event.type === 'still_need_from_others_list') {
```
Comment (austin-martinez) [inline src/js/disability-benefits/components/ClaimPhase.jsx]: Can these be added to the `switch` above?
```javascript
case 'still_need_from_you_list':
case 'still_need_from_others_list':
   return ...
case 'received_from_you_list':
case 'received_from_others_list':
  return ...
```

Comment (jbalboni) [inline src/js/disability-benefits/components/ClaimPhase.jsx]: Yep, that would be better.

**Comment by cagedcrown** [comment]
Looks good 👍 . We can also make the activity sections collapsable down the road as well, but it's not a high priority.

**Comment by carlmjohnson** [comment]
:+1:

**Comment by jbalboni** [comment]
@cagedcrown They're collapsible now, but I forgot to add the +/- icons. We can do that in another PR.

**Comment by cagedcrown** [comment]
@jbalboni Sounds good 👍

**Pull request merged**